spec/controllers/market_lists_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe MarketListsController, type: :controller do
```

Comment:
There`s a bunch of repeated code, I believe it would be better to create a shared_context... and use like this as an example

```
  shared_context "with basic setup" do
    let(:market_list1) { MarketList.create(name: 'Lista 1', market_date: Date.today) }
    let(:market_list2) { MarketList.create(name: 'Lista 2', market_date: Date.yesterday) }
  end
```
 
 And then 
 
```
   describe 'GET #index' do
    include_context "with basic setup"
```